BugReporter/BugReportForm.cs

```diff
@@ -56,8 +56,6 @@ public BugReportForm()
             exceptionTypeLabel.Image = DpiUtil.Scale(exceptionTypeLabel.Image);
             exceptionDetails.PropertyColumnWidth = DpiUtil.Scale(101);
             exceptionDetails.PropertyColumnWidth = DpiUtil.Scale(101);
-            DpiUtil.Scale(sendAndQuitButton.MinimumSize);
-            DpiUtil.Scale(btnCopy.MinimumSize);
```

Comment:
I think these either have to be
```suggestion
            sendAndQuitButton.MinimumSize = DpiUtil.Scale(sendAndQuitButton.MinimumSize);
            btnCopy.MinimumSize = DpiUtil.Scale(btnCopy.MinimumSize);
```
or
```suggestion
            DpiUtil.Scale(ref sendAndQuitButton.MinimumSize);
            DpiUtil.Scale(ref btnCopy.MinimumSize);
```